code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -20,6 +20,11 @@
 	pixel_x = rand(-3, 3)
 	pixel_y = rand(-3, 3) //randomize a little
 
+/obj/item/stack/tile/examine(mob/user)
+	..()
+	if(throwforce >= 10)
```

Comment:
I feel like there'd be a lack of nuance if at 9 throwforce it already stops being a good throwing weapon, maybe multiple options is better?

Reply:
ok